templates/CORE_DEV_WEEKLY.md

```diff
@@ -1,6 +1,7 @@
 <!-- Pick the correct title -->
 # ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 <enter date here> 
 # 💫Golang Core Dev Team Weekly Sync 🙌🏽 <enter date here>
+# 🖧 IPLD Bi-Weekly Sync 🙌🏽 <enter date here>
```

Comment:
What is this symbol? 

<img width="72" alt="image" src="https://user-images.githubusercontent.com/1211152/48644739-75e74300-e9db-11e8-8b98-b556972723e1.png">

Reply:
@vmx